_includes/spark/_allgotrips.html

```diff
@@ -1,5 +1,41 @@
 <div class="allgotrips grid">
 
+  <div class="season-4 grid-item push-ends col-xs-12 col-sm-4 col-md-3">
+    <div class="grid-item-content">
+      <a href="https://youtu.be/sRQ_DaHNoVI" target="_blank"><img
+             class="imgix-fluid img-responsive" alt="" title=""
+             data-src="//crds-media.imgix.net/6sNyLY6qD4shJXGN6Ms6ek/f53a13abd33a51f6ed4912a221a552b1/jeremy-affeldt-spark.jpg"></a>
+      <h3 class="font-family-condensed-extra text-uppercase font-size-h4 push-half-top flush-bottom"
+          style="line-height: .9em;"><a
+           href="https://youtu.be/sRQ_DaHNoVI" target="_blank">Being a Pro Athlete Couldn't Cure My Loneliness</a></h3>
+      <p class="font-size-small font-family-base push-quarter-top">Jeremy Affeldt</p>
+    </div>
+  </div>
+
+  <div class="season-4 grid-item push-ends col-xs-12 col-sm-4 col-md-3">
+    <div class="grid-item-content">
+      <a href="https://youtu.be/haPN4PV_UMg" target="_blank"><img
+             class="imgix-fluid img-responsive" alt="" title=""
+             data-src="//crds-media.imgix.net/56ZAOUZAq8YbFuKy9GCErz/a20cc16288d2179a5722d13f3913a5e8/otis-williams-spark.jpg"></a>
+      <h3 class="font-family-condensed-extra text-uppercase font-size-h4 push-half-top flush-bottom"
+          style="line-height: .9em;"><a
+           href="https://youtu.be/haPN4PV_UMg" target="_blank">The One Word That Changed My Life</a></h3>
+      <p class="font-size-small font-family-base push-quarter-top">Otis Williams</p>
+    </div>
+  </div>
+
+<div class="season-4 grid-item push-ends col-xs-12 col-sm-4 col-md-3">
+  <div class="grid-item-content">
+    <a href="https://youtu.be/GE_3IJt59yQ" target="_blank"><img
+           class="imgix-fluid img-responsive" alt="" title=""
+           data-src="//crds-media.imgix.net/2CPtJbJdJ13MaS1bSPH5cT/63eb6edb94d898c3cf913b01fcc8c9ab/kim-botto-spark.jpg"></a>
+    <h3 class="font-family-condensed-extra text-uppercase font-size-h4 push-half-top flush-bottom"
+        style="line-height: .9em;"><a
+         href="https://youtu.be/GE_3IJt59yQ" target="_blank">Where the Beatles Got It Wrong</a></h3>
+    <p class="font-size-small font-family-base push-quarter-top">Kim Botto</p>
+  </div>
+</div>
+
```

Comment:
should this be here? was this removed in a hotfix?

Reply:
This should be here. The hotfix was adding it. Jake said its g2g in.

Reply:
👍